src/main/java/aspects/MetricsAspect.java

```diff
@@ -55,9 +64,23 @@ public Object logRequest(ProceedingJoinPoint joinPoint) throws Throwable {
                 "user:" + user,
                 "request:" + requestPath
         );
+        if (durationBucket.equals("lt_120s") || durationBucket.equals("over_120s")) {
+            sendTimingWarningToSentry(timeInMs);
+        }
         return result;
     }
 
+    private void sendTimingWarningToSentry(final long timeInMs) {
+        raven.recordBreadcrumb(new BreadcrumbBuilder() {{
+            setCategory("long_request");
+            setLevel(Breadcrumb.Level.WARNING);
+            setData(new HashMap<String, String>() {{
+                put("duration", String.format("%.3fs", timeInMs / 1000.));
+            }});
+        }}.build());
```

Comment:
@wspride I like this syntax better than the more linear one, where you declare the hashmap, then put a thing into it, then declare the builder, then add things to it, including the hashmap you declared and populated above, and then finally pass `builder.build()` to raven. It's more like what I'm used to doing in python, and more clearly shows that all of these things are really just a single conceptual step of recording a breadcrumb. Do you have thoughts? (And this is supported in the java version we're using, right?)

Reply:
I don't really have a problem with this (looks pretty) but doesn't seem to be beloved by Java purists: https://stackoverflow.com/a/14878801. Given how small the scope is here I think no problem